src/Application/Common/Services/CurrencyService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using DnDCharacterSheet.Application.Common.Interfaces;
+using DnDCharacterSheet.Domain.Enums;
+using DnDCharacterSheet.Domain.ValueObjects;
+
+namespace DnDCharacterSheet.Application.Common.Services;
+public class CurrencyService : ICurrencyService
+{
+    private static readonly Dictionary<(Currencies, Currencies), int> ConversionRates = new()
```

Comment:
?

Reply:
Is there any way to improve this? I chose this because I think there is no need to make it more complex